tests/unit/Http/Request/GetClientAddressCest.php

```diff
@@ -13,20 +13,125 @@
 
 namespace Phalcon\Test\Unit\Http\Request;
 
+use Phalcon\Http\Request;
 use UnitTester;
 
 class GetClientAddressCest
 {
+    /**
+     * Tests Phalcon\Http\Request :: getClientAddress() - trustForwardedHeader
+     *
+     * @author Phalcon Team <team@phalcon.io>
+     * @since  2020-03-17
+     */
+    public function httpRequestGetClientAddressTrustForwardedHeader(UnitTester $I)
+    {
+        $I->wantToTest('Http\Request - getClientAddress() - trustForwardedHeader');
+
+        $store   = $_SERVER ?? [];
+        $time    = $_SERVER['REQUEST_TIME_FLOAT'];
+        $_SERVER = [
+            'REQUEST_TIME_FLOAT'   => $time,
+            'HTTP_X_FORWARDED_FOR' => '10.4.6.1',
+        ];
+
+        $request = new Request();
+
+        $expected = '10.4.6.1';
```

Comment:
Can we add a ipv6 test case as well?

Reply:
It's a bit irrelevant to be honest. The address comes form `$_SERVER['REMOTE_ADDR']` so I can put banana in there and it will be returned.

Added one more test just for that

Reply:
Thnx. Just to be sure.